modules/howtos/pages/logging.adoc

```diff
@@ -13,84 +13,65 @@ The Columnar Go SDK offers simple logging of library internals to help debug iss
 ----
 
-It is also possible to provide other logger implementations to [.api]`gocb.SetLogger`.
-Implementations must satisify the [.api]`gocb.Logger` interface.
+It is also possible to provide other logger implementations to [.api]`SetLogger`.
+Implementations must satisify the [.api]`cbanalytics.Logger` interface.
 
 [source,golang,indent=0]
 ----
-  type Logger interface {
-	// Outputs logging information:
-	// level is the verbosity level
-	// offset is the position within the calling stack from which the message
-	// originated. This is useful for contextual loggers which retrieve file/line
-	// information.
-	Log(level LogLevel, offset int, format string, v ...interface{}) error
-}
-----
-
-The [.api]`gocb.DefaultStdioLogger()` and [.api]`gocb.VerboseStdioLogger()` wrap their [.api]`gocbcore` counterparts to provide a stable interface.
-The [.api]`gocb` versions should be used.
+type Logger interface {
+	// Error outputs an error level log message.
+	Error(format string, v ...interface{})
 
+	// Warn outputs an warn level log message.
+	Warn(format string, v ...interface{})
 
-== Log Redaction
+	// Info outputs an info level log message.
+	Info(format string, v ...interface{})
 
-Redacting logs is a two-stage process. 
-If you want to redact client logs (for example before handing them off to the Couchbase Support team) you first need to enable log redaction in your application.
+	// Debug outputs a debug level error log message.
+	Debug(format string, v ...interface{})
 
-[source,golang,indent=0]
-----
-gocb.SetLogRedactionLevel(gocb.RedactFull)
+	// Trace outputs a trace level error log message.
+	Trace(format string, v ...interface{})
+}
 ----
 
-Different redaction levels are supported -- please see the `RedactionLevel` enum description for more information.
-
-Note that you need to run this command before any of the SDK code is initialized so all of the logs are captured properly. 
-Once the SDK writes the logs with the tags to a file, you can then use the xref:7.1@server:cli:cbcli/cblogredac
-
-* You may wish to read more on Log Redaction xref:7.1@server:manage:manage-logging/manage-logging.adoc#understanding_redaction[in the Server docs].
-
 == Using your own Logger
 
 Sometimes you want to use your own logger with the SDK.
 You might want your logging to use a popular logging framework such as logrus.
 In the following examples we show to use the SDK with a logrus logger:
 
 First we need to create our own logger that wraps the logrus logger.
-The logrus `Log`/`Logf` functions don't quite match the gocb logging interface and the log levels are slightly different.
+The logrus logging functions don't quite match the gocbanalytics logging interface.
 This means that we need to do a bit of marshalling to get the data into a set of parameters that logrus can use.
 
 [source,golang,indent=0]
 ----
-include::go-sdk:devguide:example$go/custom-logging.go[tag=loggerwrapper]
+include::devguide:example$logging.go[tags=loggerwrapper]
```

Comment:
While this should still work, a single excerpt only needs `tag`
https://docs.asciidoctor.org/asciidoc/latest/directives/include-tagged-regions/#tagging-regions